webapp/src/components/cardDialog.tsx

```diff
@@ -129,25 +132,86 @@ const CardDialog = (props: Props): JSX.Element => {
         </CardActionsMenu>
     )
 
-    const followActionButton = (following: boolean): React.ReactNode => {
-        const followBtn = (
+    const handler = contentRegistry.getHandler('attachment')
+    if (!handler) {
+        Utils.logError('addContentMenu, unknown content type: attachment')
+        return <></>
+    }
+
+    const addElement = async () => {
+        if (card) {
+            const block = await handler.createBlock(card.boardId, intl)
+            block.parentId = card.id
+            block.boardId = card.boardId
+            const typeName = handler.getDisplayText(intl)
+            const description = intl.formatMessage({id: 'ContentBlock.addElement', defaultMessage: 'add {type}'}, {type: typeName})
+            await mutator.performAsUndoGroup(async () => {
+                const afterRedo = async (newBlock: Block) => {
+                    const contentOrder = card.fields.contentOrder.slice()
+                    contentOrder.splice(card.fields.contentOrder.length, 0, newBlock.id)
+                    await octoClient.patchBlock(card.boardId, card.id, {updatedFields: {contentOrder}})
+                }
+
+                const beforeUndo = async () => {
+                    const contentOrder = card.fields.contentOrder.slice()
+                    await octoClient.patchBlock(card.boardId, card.id, {updatedFields: {contentOrder}})
+                }
+
+                await mutator.insertBlock(block.boardId, block, description, afterRedo, beforeUndo)
+            })
+        }
+    }
+
+    const deleteBlock = useCallback(async (block: Block) => {
```

Comment:
Per spec, delete should have confirmation.

Reply:
Added